apps/backend/prisma/client/sql/subscriberGrowthQuery.d.ts

```
@@ -1,23 +1,16 @@
  * @param timestamp
  * @param timestamp
  */
-export const subscriberGrowthQuery: (
-  text: string,
-  timestamp: Date,
-  timestamp: Date,
-) => $runtime.TypedSql<
-  subscriberGrowthQuery.Parameters,
-  subscriberGrowthQuery.Result
->;
+export const subscriberGrowthQuery: (text: string, timestamp: Date, timestamp: Date) => $runtime.TypedSql<subscriberGrowthQuery.Parameters, subscriberGrowthQuery.Result>
```

Biome (1.9.4)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)